packages/@vue/cli-test-utils/createTestProject.js

```diff
@@ -17,7 +17,7 @@ module.exports = function createTestProject (name, preset, cwd, initGit = true)
     return fs.existsSync(path.resolve(projectRoot, file))
   }
 
-  if (has(projectRoot)) {
+  if (has(name)) {
```

Comment:
I was getting
```
console.warn packages/@vue/cli-test-utils/createTestProject.js:21
    An existing test project already exists for e2e-serve. May get unexpected test results due to project re-use
```
warnings, so I guess this should have been `name` instead?

Reply:
Nvm, wrong assumption on my side, the issue seems to be that the test project are not cleaned up after. Should I also fix that in this PR?

Reply:
Use a different name for the test project would suffice 😅

Reply:
I'm not sure I understand what you mean. 🙂 The issue is that tests create test projects in `packages/test` when they run (e.g. `packages/test/e2e-serve`), which don't get cleaned up, so next time I run tests I get the warning I mentioned above about them already existing:
```
  console.warn packages/@vue/cli-test-utils/createTestProject.js:22
    An existing test project already exists for e2e-serve D:\work\vue-stuff\vue-cli\packages\test\e2e-serve. May get unexpected test results due to project re-use
```

Reply:
When running tests locally it can be solved by running `yarn clean` first.
The warning in the CI environment is because this test case shares the same project name as the first test case in that file (both are `e2e-serve`).

Reply:
Oh, got it, I was talking more about the warnings I got while working on this, I didn't even saw the warning in CI. 🙂 So should I go ahead and change the project name for the test I added?

Reply:
Yeah I think so

Reply:
Sure! Any other feedback, so that I can address everything in a single commit?

Reply:
The rest of this PR looks pretty good to me 😄